docs/content/config/security/mail_crypt.md

```diff
@@ -59,8 +65,49 @@ Official Dovecot documentation: https://doc.dovecot.org/configuration_manual/mai
 
 6. Start the container, monitor the logs for any errors, send yourself a message, and then confirm the file on disk is encrypted:
     ```
-    [root@ip-XXXXXXXXXX ~]# cat -A /mnt/efs-us-west-2/maildata/awesomesite.com/me/cur/1623989305.M6v�z�@�� m}��,��9����B*�247.us-west-2.compute.inE��\Ck*�@7795,W=7947:2,
-    T�9�8t�6�� t���e�W��S   `�H��C�ڤ �yeY��XZ��^�d�/��+�A
+    root@ip-xxx:/# xxd /var/mail/xxx/xxx/cur/1625835047.M216159P15117.ip-xxx\,S\=80516\,W\=81800\:2\,S  | head
+    00000000: 4352 5950 5445 4403 0702 0000 0002 0000  CRYPTED.........
+    00000010: 0143 0609 6086 4801 6503 0401 2e06 0960  .C..`.H.e......`
+    00000020: 8648 0165 0304 0201 0000 0800 0000 0113  .H.e............
+    00000030: 0102 39f8 ca52 927f ff3d 5be1 0a7e 52e0  ..9..R...=[..~R.
+    00000040: 6ae0 e40e 8584 5f89 851b 076c 39c0 14ed  j....._....l9...
+    00000050: 8b21 0000 0085 0401 0288 970f d348 6460  .!...........Hd`
+    00000060: 29f7 66e4 60f7 57f3 e293 cf1e 4364 7ad7  ).f.`.W.....Cdz.
+    00000070: 7d29 0fe5 6cf7 5666 4aa1 b4c9 ddcf 650a  })..l.VfJ.....e.
+    00000080: 193b 7f31 072a 28ad d84b c038 ddd2 f060  .;.1.*(..K.8...`
+    00000090: 9931 7282 4d4e dd51 e101 9406 d4fc e3d0  .1r.MN.Q........
     ```
 
 This should be the minimum required for encryption of the mail while in storage.
+
+
+## Encrypted Per User Keys 
+
+Enables per user storage encryption keys and password protects them.
+
+1. Set `ENABLE_PER_USER_STORAGE_ENCRYPTION=1` in your docker-compose.yml
+
+2. Restart the mailserver with `docker-compose down` and `up`
+
+!!! info
+ 
+    You can change the mail_crypt_curve used by setting `PER_USER_STORAGE_ENCRYPTION_CURVE`, the default value is `secp521r1`
```

Comment:
I am missing this var in the [mailserver.env](https://github.com/docker-mailserver/docker-mailserver/blob/master/mailserver.env)

Reply:
I forgot to add that 😅

Reply:
https://github.com/docker-mailserver/docker-mailserver/pull/2080/commits/4efccbbc72ebef2a2a45efa3e5c2d9725d9eafe0